lib/voyager/clients/twitter_client.rb

```diff
@@ -183,6 +183,18 @@ def recent_tweets(options={})
       get("/statuses/user_timeline.json", options)
     end
 
+    # ============================================================================
+    # Metrics Methods - These act on Tweets
+    # ============================================================================
+
+    def follower_count(options = {})

```

Comment:
the pattern we use here is to have the id be explicit (we call it `user_id` in the `follower_ids` method, so let's be consistent there). `def follower_count(user_id, options={})`